manifests/overlays/rhoai/params.env

```diff
@@ -0,0 +1 @@
+odh-dashboard-image=quay.io/opendatahub/odh-dashboard:main
```

Comment:
so the file is change from 	`manifests/overlay/rhods/params.env`
to `manifests/overlays/rhoai/params.env`

we will need am update in operator for this

Reply:
Yess! That's one of the things I was gonna ping you about, I'm doing the list of changes for the platform to be aware of!